client/src/components/assessments/AssessmentResultsTable.js

```diff
@@ -142,7 +142,9 @@ const AssessmentResultsTable = ({
       {showAssessmentResults && (
         <div style={{ ...style }}>
           <Fieldset
-            title={`Assessment results - ${recurrence}`}
+            title={`Assessment results - ${
+              assessmentConfig.label || recurrence
+            }`}
```

Comment:
This means that an admin shouldn't use the same label for two assessments with different recurrence, so maybe document that somewhere? (not sure what would be the proper place for that though)

Reply:
I think the description field in the _anet-schema.yml_ is a good place to mention that so I put some explanation there.